### PR TITLE
Thomas Wilson Chunk #1

### PR DESCRIPTION
**Pull request by kiahjh**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal-letters/modernized/03-epistles.adoc [hunk 1]
```diff
@@ -1,8 +1,13 @@
 == Epistles Written by Thomas Wilson, And His Fellow-Laborer in the Service of the Gospel, James Dickinson, While in America.
```
Comment (Henderjay) [inline journal-letters/modernized/03-epistles.adoc:2]: This chapter heading is too long for it to fit as a "header" on a book page. So you need to make a shortened version, maybe to _Epistles of Thomas Wilson and James Dickinson_ . To do this, look under the Short Titles section on the style guide.👍🏻

File: journal-letters/modernized/04-testimonies.adoc [hunk 2]
```diff
@@ -95,11 +96,13 @@ and touch the tongues of many of our youth, with a live coal from his holy altar
 that many may be willing to run his errand, and be serviceable in his hand,
 as were many of the generation, whom he has removed from us.
 
-Mary Wilson.
+[.signed-section-signature]
+Mary Wilson
 
 [.asterism]
 '''
```
Comment (Henderjay) [inline journal-letters/modernized/04-testimonies.adoc]: I think we can remove this asterism here. Come up to my office and I'll try to explain when we keep, and when we kill the asterisms.

File: journal-letters/modernized/04-testimonies.adoc [hunk 3]
```diff
@@ -149,13 +152,16 @@ so as to obtain the blessing while here,
...
 when time to us shall be no more.
 
-Thomas Wilson, Benjamin Wilson.
+[.signed-section-signature]
+Thomas Wilson, Benjamin Wilson
 
+[.signed-section-context-close]
 Thornwell, the 1st of the Third month, 1727.
 
 [.asterism]
```
Comment (Henderjay) [inline journal-letters/modernized/04-testimonies.adoc]: I think we can kill this one too.

File: journal-letters/modernized/04-testimonies.adoc [hunk 4]
```diff
@@ -298,14 +304,15 @@ concerning his faithful and diligent labors in the work of the Gospel,
...
 which wrought effectually in him, and whereof he was a faithful witness,
-to whom be glory forever.
-Amen.
+to whom be glory forever. Amen.
 
-James Dickinson.
+[.signed-section-signature]
+James Dickinson
 
 [.asterism]
```
Comment (Henderjay) [inline journal-letters/modernized/04-testimonies.adoc]: and this one

File: journal-letters/modernized/04-testimonies.adoc [hunk 5]
```diff
@@ -370,22 +377,27 @@ a living spring of the ministry, and that many may be made willing to run his er
...
 Joshua Clibborn, John Barclay, Mungo Bewley, Gregory Russel, Joseph Inman, John Stoddart,
 Jonathan Robinson, Edward Barwick, Joseph Gill, John Boles, Richard Sealey,
 Abel Strettel, Thomas Ducrett, Solomon Watson, Ralph Stephenson, Samuel Watson,
-Nicholas Harris, James Pillar, Bartho.
-Garnett, Thomas Greer, James Hill, Daniel Bewley, Toby Courtney, Samuel Fuller,
-John Chrysty.
+Nicholas Harris, James Pillar, Bartho Garnett, Thomas Greer, James Hill, Daniel Bewley,
+Toby Courtney, Samuel Fuller, John Chrysty.
 
 [.asterism]
```
Comment (Henderjay) [inline journal-letters/modernized/04-testimonies.adoc]: kill this one too.

File: journal-letters/modernized/03-epistles.adoc [hunk 6]
```diff
@@ -95,11 +100,16 @@ So remain your friends and brethren in the truth, letting you know that we are w
 with love to you all, sending these lines as a token thereof.
 Farewell.
 
+[.signed-section-signature]
 Thomas Wilson, James Dickinson.
 
-=== An Epistle to Friends in Pennsylvania.
+[.chapter-subtitle--blurb]
```
Comment (Henderjay) [inline journal-letters/modernized/03-epistles.adoc]: I think I told you wrong on this.  Sorry!!! I think this the remaining letter headings in this chapter should be just `[.blurb]` tags, and not `[.chapter-subtitle-blurb]`

File: journal-letters/modernized/03-epistles.adoc [hunk 7]
```diff
@@ -194,12 +204,16 @@ all fear, obedience and worship, both now and evermore.
...
 
-Thomas Wilson.James Dickinson.
+[.signed-section-signature]
+Thomas Wilson, James Dickinson.
 
+[.signed-section-context-close]
 From Rhode Island, the 5th of the Fifth month, 1692.
 
-=== An Epistle to Friends in Barbados and Elsewhere.
+[.chapter-subtitle--blurb]
```
Comment (Henderjay) [inline journal-letters/modernized/03-epistles.adoc]: so make this one a `[.blurb]` too

File: journal-letters/modernized/03-epistles.adoc [hunk 8]
```diff
@@ -330,13 +344,19 @@ Amen.
...
 We remain your friends and brethren in the truth,
 
+[.signed-section-signature]
 Thomas Wilson, James Dickinson.
 
+[.signed-section-context-close]
 From Antigua, the 14th of the Tenth month, 1692.
 
-=== An Epistle to Friends in Philadelphia.
+[.chapter-subtitle--blurb]
```
Comment (Henderjay) [inline journal-letters/modernized/03-epistles.adoc]: just blurb here too.

File: journal-letters/modernized/03-epistles.adoc [hunk 9]
```diff
@@ -392,13 +412,19 @@ that you may be kept by the power of God, unto whom we leave you,
...
 wherein we remain your friends and brethren,
 
+[.signed-section-signature]
 Thomas Wilson, James Dickinson.
 
+[.signed-section-context-close]
 Nevis, the 4th of the Twelfth month, 1692-3.
 
-=== Another Epistle to Friends in America.
+[.chapter-subtitle--blurb]
```
Comment (Henderjay) [inline journal-letters/modernized/03-epistles.adoc]: and just blurb here.

File: journal-letters/modernized/03-epistles.adoc [hunk 10]
```diff
@@ -441,13 +467,16 @@ and ease the spirits of them that travel under the weight thereof.
 Unto him we are willing to commit the cause, and do commit you, and our own souls;
 so remain your friends and brethren,
 
+[.signed-section-signature]
 Thomas Wilson, James Dickinson.
 
-=== To Friends of Mount-melick, Men`'s Meeting, and Elsewhere.
+[.chapter-subtitle--blurb]
```
Comment (Henderjay) [inline journal-letters/modernized/03-epistles.adoc]: and blurb here.

File: journal-letters/modernized/04-testimonies.adoc [hunk 11]
```diff
@@ -406,6 +418,7 @@ to the refreshing and edifying of Friends.
...
```
Comment (Henderjay) [inline journal-letters/modernized/04-testimonies.adoc]: you can kill this asterism

File: journal-letters/modernized/04-testimonies.adoc [hunk 12]
```diff
@@ -445,6 +458,7 @@ wherein the Lord blessed his endeavors with prosperity and plenty.
...
```
Comment (Henderjay) [inline journal-letters/modernized/04-testimonies.adoc]: and kill this one

File: journal-letters/modernized/04-testimonies.adoc [hunk 13]
```diff
@@ -465,6 +479,7 @@ have been given forth in the wisdom of truth.
...
```
Comment (Henderjay) [inline journal-letters/modernized/04-testimonies.adoc]: and kill this one

File: journal-letters/modernized/04-testimonies.adoc [hunk 14]
```diff
@@ -515,6 +530,7 @@ Samuel Peacock, John Scott.
...
```
Comment (Henderjay) [inline journal-letters/modernized/04-testimonies.adoc]: and kill this one

File: journal-letters/modernized/04-testimonies.adoc [hunk 15]
```diff
@@ -591,9 +607,10 @@ Nathan Newby.
...
```
Comment (Henderjay) [inline journal-letters/modernized/04-testimonies.adoc]: and kill this one

File: journal-letters/modernized/04-testimonies.adoc [hunk 16]
```diff
@@ -650,13 +667,16 @@ who has done his day`'s work carefully, and is entered into that rest,
...
 yet his memory will live amongst us.
 
-George Marke.
+[.signed-section-signature]
+George Marke
 
+[.signed-section-context-close]
 Mosedale, in Cumberland, the 28th of the Third month, 1726.
 
 [.asterism]
```
Comment (Henderjay) [inline journal-letters/modernized/04-testimonies.adoc]: and kill this one

**Review by Henderjay — CHANGES_REQUESTED** [review]
Really good work, Kiah. The main things I want you to change here are to remove most of the asterisms (where marked), and to change some of the headings (that I might have told you wrong).  But none of these things are really _mistakes_... they are just things you need to learn.  You did a really good and careful job! Fix the stuff I marked, and I'll merge it.  Oh... and come up to my office to have me explain to you how we use the asterisms in these final copies.👍🏻

**Comment by kiahjh** [comment]
Okay, I’ll come up soon!

-Kiah

**Review by Henderjay — CHANGES_REQUESTED** [review]
The only thing to do is kill your one comment.

**Comment by Henderjay** [comment]
looks great, merging!!

**Comment by kiahjh** [comment]
Okay.

-Kiah

**Pull request merged**